easybuild/easyconfigs/p/packmol/packmol-v20.2.2-intel-2020a.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'packmol'
+version = 'v20.2.2'
+
+homepage = 'http://m3g.iqm.unicamp.br/packmol'
+description = "Packing Optimization for Molecular Dynamics Simulations"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
```

I think packmol can be safely moved to `iccifort` toolchain
```suggestion
toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
```

---

added